### ISSUE TITLE
Detailed view of listed property

### ISSUE THREAD

**Issue by kzcheng** [CMPT276-Project-Group-6/Realtorest#69]
When the user clicks on a listing in the property listings view, there should be more details for the property being shown.

**Comment by kzcheng** [comment]
Nam plans to work on implementing the details views of property listings in the UI. Before Kevin is done with the API stuff, Nam will just use mock data. After Kevin is done, someone will need to switch the UI to use actual data received from Realtor.ca instead.

Nam will likely be able to complete this before the end of Friday 22nd.

**Comment by namneyugn21** [comment]
I was aiming to have the detailed view to be on a different page, so that user can have a better view and it is more aesthetic pleasing to be like that. However, the problem is that I think it's difficult to export the data from a javascript file to another so I think I will just do a pop up Modal to display the details 👯

**Comment by kzcheng** [comment]
> I was aiming to have the detailed view to be on a different page, so that user can have a better view and it is more aesthetic pleasing to be like that. However, the problem is that I think it's difficult to export the data from a javascript file to another so I think I will just do a pop up Modal to display the details 👯

Good idea!

I think both should options work fine.

If there is any help you need, let me know.

**Comment by namneyugn21** [comment]
![Screenshot 2024-03-21 at 4 20 04 PM](https://github.com/CMPT276-Project-Group-6/Realtorest/assets/108586314/6b5c8a72-701d-4d1d-84bd-178efdfb245e)
![Screenshot 2024-03-21 at 4 20 09 PM](https://github.com/CMPT276-Project-Group-6/Realtorest/assets/108586314/e7442400-acc5-4152-9a1b-82c2a6e70676)

**Comment by namneyugn21** [comment]
Here is the modal that contains the detailed description of the property. 
Key highlights:
- The carousel image will automatically change the picture every 3 seconds (more or less :)))
- The picture changing effect will be 'Crossfade', and this has been applied to be the standard. 
- I removed the star icon represent 'Add to favourites'. Instead, user can click on the bookmark icon to save to their favourite. This is more convenient, as it is hard to see the star on the picture and hard to manipulate it via Z-Index in CSS.
- Finally, in the future, we will definitely add the map below the description when I got to the API part.
